lab.js

```diff
@@ -0,0 +1,13 @@
+document.addEventListener("DOMContentLoaded", () => {
+    let button = document.querySelector("#reset")
```

Comment:
This looks good but lets try to have a button that toggles between having a style and not having one.